db/airports.json

```diff
@@ -0,0 +1,122 @@
+{
+	"airport1": {
+		"uid": "Sarah",
+		"airportName": "Kansas City International Airport",
+    "imgUrl": "https://images.fineartamerica.com/images/artworkimages/mediumlarge/2/kc-skyline-kansas-city-downtown-and-union-station-gregory-ballos.jpg",
+		"city": "Kansas City",
+		"state": "MO",
+		"country": "US",
+		"airportCode": "MCI",
+    "isInternational": false
+	},
+	"airport2": {
+		"uid": "Stephen",
+		"airportName": "Dubai International",
+		"imgUrl": "https://gapyearsite.wpengine.com/wp-content/uploads/2017/11/articles/DubaiNight1.jpg",
+		"city": "Dubai",
+		"state": "N/A",
+		"country": "UAE",
+		"airportCode": "DXB",
+    "isInternational": true
+	},
+	"airport3": {
+		"uid": "Kenny",
+		"airportName": "John F. Kennedy Airport",
+		"imgUrl": "https://cdn11.bigcommerce.com/s-45hj43/images/stencil/500x659/products/4559/6793/CW_808_Statue_of_Liberty_with_New_York_Skyline__35635.1557789140.jpg?c=2&imbypass=on",
+		"city": "New York City",
+		"state": "NY",
+		"country": "US",
+		"airport code": "JFK",
```

Comment:
this should be camel case = airportCode

Reply:
I so appreciate you pulling EVERY DETAIL I missed..